README.rst

```diff
@@ -53,6 +53,16 @@ Installation
 
        $ pip install aws-encryption-sdk
 
+Troubleshooting
+===============
+
+Ensure you have the most recent version of pip on your system:
```

Comment:
It's not directly about having the most recent version of pip but about having the most recent version of `pyca/cryptography` and `cffi`. Getting past a certain version of each of those requires a minimum version of pip.

Reply:
This info should be in the README. Tell people when they can prevent a problem. Matt, if you can write a bullet point or two, I'll make it into words for the readme.